lib/libc/sys/cheri_revoke.2

```diff
@@ -52,14 +53,13 @@ invokving the
 system call.
 .Pp
 The
-.Fn cheri_revoke_shadow
-system call is used to obtain access to segments of the shadow bitmap
-and to global revocation metadata.
```

Comment:
For better or worse, this syscall is still used to access the epoch counter.  If we had per-AS VDSO data, that'd doubtless be the better way to do it.

Reply:
Oh, yes, I shouldn't have deleted this.